app/models/viewer.rb

```diff
@@ -17,6 +17,8 @@ def default_viewer_path
   def to_partial_path
     return default_viewer_path if viewer_type == 'sul-embed' || viewer_type.nil?
 
-    "../viewers/#{viewer_type}"
+    modified_viewer_type = viewer_type == 'mirador' ? 'mirador3' : viewer_type
```

Comment:
We could fallback a different way.. open to suggestions here.

Reply:
This seems... reasonable enough for now? Could we throw a data remediation on the pile for our next maintenance work cycle?

Reply:
Forgot to mention, I just checked in prod (about an hour ago) and no more Exhibits had this.. that is why I just went with this.